react-nginx/Dockerfile

```diff
@@ -22,6 +24,22 @@ FROM development AS build
 
 RUN npm run build
 
+
+FROM development as dev-envs
```

Comment:
Is `dev-envs` a magic name for a target? And all these commands? Wish there was a way to avoid copy & paste.

Reply:
`dev-envs` isn't a magic name, it's just a convention we use for all the samples.
I understand your wish, meanwhile the goal of this repo is to give the whole information needed to start a specific stack. We may assume that most of the users will only check 1 or 2 samples so we need to give them the whole recipe to properly configure their stack to work with Development Environments feature. 
WDYT?